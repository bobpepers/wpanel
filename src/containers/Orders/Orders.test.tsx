import { shallow } from 'enzyme';
import * as React from 'react';
import { Orders } from './';

describe('Activities test', () => {
    it('should render', () => {
        const wrapper = shallow(<Orders />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
