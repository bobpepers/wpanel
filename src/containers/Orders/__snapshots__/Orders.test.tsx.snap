// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Activities test should render 1`] = `<Fragment />`;
